crates/hc/src/bin/hc.rs

```diff
@@ -103,29 +103,32 @@ async fn main() -> anyhow::Result<()> {
                 hc::force_admin_port(path, port)?;
             }
             if let Some(ports) = run {
-                run_n(&ops.holochain_path, paths, ports, ops.force_admin_ports).await?;
+                let holochain_path = ops.holochain_path.clone();
```

Comment:
Inlining `run_n` twice seems like a move in a wrong direction. Though previous name doesn't seem all that great either.

Reply:
Hi, this isn't actually inlined. It's just spawning the run so we can wait on ctrl_c and cleanup the file locks.

Reply:
What I mean by "inlinining" is a call to `run_n` turned into ~15LoC here in the place below. Maybe both of these should be `run_n_locked` or something?